File: src/dltpgh/baekjoon/_2587.py
num_list = []
for i in range(5): # 5개의 자연수 반복문 출력
    a = int(input())
    num_list.append(a) # 입력된 자연수 리스트에 저장
num_list.sort() # 리스트 정렬
print(int((sum(num_list) / 5))) # 평균값 출력
print(num_list[2]) # 중앙값 출력

File: src/zeroradish/baekjoon/_2587.java
package zeroradish.baekjoon;

import java.util.Arrays;
import java.util.Scanner;

public class _2587 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int size = 5;
        int[] Ns = new int[size];

        for(int i = 0; i< Ns.length; i++){
            int N = sc.nextInt();
            Ns[i] =  N;
        }
        Arrays.sort(Ns);
        System.out.println(Arrays.stream(Ns).sum()/size);
        System.out.println(Ns[2]);
    }
}
